internal/clientcache/internal/cache/status.go

```diff
@@ -125,7 +125,7 @@ func (s *StatusService) Status(ctx context.Context) (*Status, error) {
 			us.AuthTokens = append(us.AuthTokens, *ts)
 		}
 
-		for _, rt := range []resourceType{targetResourceType, sessionResourceType} {
```

Comment:
Looks like the status_test needs to be updated to account for this change as it's failing now:
```
--- FAIL: TestStatus (0.01s)
  --- PASS: TestStatus/empty_status (0.00s)
   --- FAIL: TestStatus/users_with_no_resources (0.00s)
   --- FAIL: TestStatus/users_with_errors (0.00s)
```